openslides/core/static/js/core/projector.js

```diff
@@ -150,7 +150,8 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
     'slides',
     'Config',
     'ProjectorID',
-    function($scope, $location, $timeout, Projector, slides, Config, ProjectorID) {
+    'Logos',
```

Comment:
@normanjaeckel The Logos basically use the config. I would prefer moving this `Logo` factory in the core/base.js. Only the changing happens in the mediafiles, so mediafiles is a dependency of core and not the other way around.